src/vs/workbench/contrib/terminal/browser/xterm/xtermTerminal.ts

```diff
@@ -281,7 +289,7 @@ export class XtermTerminal extends DisposableStore implements IXtermTerminal, II
 		});
 	}
 
-	focusAccessibleBuffer(): void {
+	async focusAccessibleBuffer(): Promise<void> {
 		this._accessibileBuffer?.focus();
 	}
```

Comment:
Was this an accident? Doesn't need to be async

Reply:
It wasn't an accident

```ts
const model = await this._getTextModel(URI.from({ scheme: ACCESSIBLE_BUFFER_SCHEME, fragment }));
```